source/gent/tier1_hortense.rst

```diff
@@ -350,6 +357,33 @@ Trying to make any changes to files that are accessed via ``/readonly`` will res
    visible via ``/readonly`` during the lifetime of the job, so you should not assume that this will be the case.
 
 
+.. _hortense_interactive_debug:
+
+Interactive and debug partition
+*******************************
+
+A (small) interactive and debug partition `debug_rome` is available where you can get
+quick access but only to a limited number of resources.
```

Comment:
I would clarify what "limited number of resources" means (max. 8 cores in use, max N jobs running + X jobs in queue, etc.).

That duplicates some info from https://docs.hpc.ugent.be/interactive_debug, but it's important to highlight that imho.